flake.nix

```diff
@@ -48,11 +48,11 @@
         # duckdb
         duckdb
         # mysql
-        mycli
+        mariadb-client
         # pyspark
         openjdk17_headless
         # postgres client
-        pgcli
+        postgresql
```

Comment:
The previous commands are implemented in python, and have a dependency on an old sqlglot which conflicts with `poetry.lock`, so I moved us back to the less-nice-to-use CLI tools.